exporter/tests/test_view.py

```diff
@@ -37,67 +65,45 @@ def login_user(self):
 
     def test_csv_export(self):
         """
-        Test GET route /campaigns/export/<slug-value>/ (campaign)
+        Test GET route /campaigns/exportCSV/<slug-value>/ (campaign)
         """
-        self.login_user()
 
-        asset = create_asset()
+        campaign_slug = "test-campaign"
 
         response = self.client.get(
-            reverse(
-                "transcriptions:export-csv", args=(asset.item.project.campaign.slug,)
-            )
+            reverse("transcriptions:export-csv", args=(campaign_slug,))
         )
 
         self.assertEqual(response.status_code, 200)
+        response_content = ""
```

Comment:
Couldn't this be `"".join(response.streaming_content)`?

Reply:
I don't think so. I tried that and this is what I got:
```
Traceback (most recent call last):
  File "/Users/rstorey/workspace/concordia/exporter/tests/test_view.py", line 67, in test_csv_export
    response_content = "".join(response.streaming_content)
TypeError: sequence item 0: expected str instance, bytes found
```

Reply:
Add the `b` prefix to mark it as bytes (`b"".join(…)`) or `"".join(map(str, response.streaming_content))` but I normally prefer to keep bytes as bytes unless we're explicitly decoding them into text.

Reply:
Trying the `b"".join()` method doesn't work because you can't mix bytes and string literals.